### PR TITLE
Close the app after the user acknowledges the Launch Error alert.

### PR DESCRIPTION
**Pull request by tvolkert**

### CODE REVIEW

**Comment by tvolkert** [comment]
@chinmaygarde

File: sky/shell/platform/ios/framework/Source/FlutterViewController.mm [hunk 1]
```diff
@@ -153,20 +163,21 @@ - (void)connectToEngineAndLoad {
...
-                        }];
+  [_dartProject
+      launchInEngine:_engine
+      embedderVMType:type
+              result:^(BOOL success, NSString* message) {
+              if (!success) {
+                UIAlertView* alert = [[UIAlertView alloc]
+                        initWithTitle:@"Launch Error"
+                              message:message
+                             delegate:[[LaunchErrorDelegate alloc] init]
```
Comment (chinmaygarde) [inline sky/shell/platform/ios/framework/Source/FlutterViewController.mm]: You are leaking the delegate. Besides, you don't need one anyway. You can create a class extension to `FlutterViewController` that implements the `UIAlertViewDelegate`, add the method in its body and specify `self` as the delegate.

Comment (chinmaygarde) [inline sky/shell/platform/ios/framework/Source/FlutterViewController.mm]: Class extensions:
```
@interface FlutterViewController() <UIAlertViewDelegate>
@end
```

Comment (tvolkert) [inline sky/shell/platform/ios/framework/Source/FlutterViewController.mm]: Done - ptal

File: sky/shell/platform/ios/framework/Source/FlutterViewController.mm [hunk 2]
```diff
@@ -141,6 +144,11 @@ - (void)setupNotificationCenterObservers {
 
 #pragma mark - Initializing the engine
 
+- (void)alertView:(UIAlertView*)alertView
+clickedButtonAtIndex:(NSInteger)buttonIndex {
```
Comment (chinmaygarde) [inline sky/shell/platform/ios/framework/Source/FlutterViewController.mm]: Man, sometimes clang-format just messes in weird ways.

**Comment by chinmaygarde** [comment]
lgtm

**Pull request merged**